backend/src/main/java/com/github/backend/controller/UserController.java

```diff
@@ -1,28 +1,40 @@
 package com.github.backend.controller;
-
+import com.github.backend.models.Boulder;
 import com.github.backend.models.User;
 import com.github.backend.models.UserDto;
+import com.github.backend.service.BoulderService;
 import com.github.backend.service.UserService;
 import lombok.RequiredArgsConstructor;
 import org.springframework.security.core.annotation.AuthenticationPrincipal;
 import org.springframework.security.oauth2.core.user.OAuth2User;
 import org.springframework.web.bind.annotation.*;
 
+import java.util.List;
 import java.util.Optional;
 
 @RestController
 @RequestMapping("/api/user")
 @RequiredArgsConstructor
 public class UserController {
     private final UserService userService;
-
+    private final BoulderService boulderService;
     @GetMapping
     public Optional<User> getUserById(@AuthenticationPrincipal OAuth2User user){
         return userService.getUserById(user.getAttributes().get("id").toString());
     }
+    @GetMapping("/myFavorites")
```

Comment:
```suggestion
    @GetMapping("/my-favorites")
```

for links you dont use camelcase :-)